projects/experimental/components/pdf-viewer/pdf-viewer.style.less

```diff
@@ -0,0 +1,81 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+tui-pdf-viewer {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    flex-direction: column;
+    inline-size: 100vw;
+    block-size: 100%;
+    flex: 1;
+    box-sizing: border-box;
+    color: var(--tui-text-primary);
+
+    --t-header: 3.5rem;
+
+    tui-root:not(._mobile) & {
+        background: #000;
+    }
+
+    tui-root:not(._mobile) tui-dialog:not([new]) & {
+        block-size: 100vh;
+    }
+
+    tui-root:not(._mobile) tui-dialog[new] & {
+        margin: -6rem calc(-1 * calc(50vw - var(--tui-width) / 2)) -2rem;
+    }
+
+    tui-root._mobile tui-dialog & {
+        --t-header: 2.5rem;
+
+        margin: 0 -1rem -1rem;
+
+        tui-app-bar {
+            margin: calc(-1 * env(safe-area-inset-top)) 0 0;
+        }
+    }
+
+    tui-sheet-dialog & {
+        --t-header: 4.5rem;
+
+        margin: 0 -1rem -1.5rem;
+
+        tui-app-bar {
+            margin: -1rem 0 0;
+        }
+    }
+
+    .t-header {
+        display: flex;
+        gap: 0.5rem;
+        align-items: center;
+        block-size: var(--t-header);
+        padding: 0 0.75rem 0 1rem;
+    }
+
+    .t-title {
+        margin: 0;
+        font: var(--tui-font-text-m);
+        white-space: nowrap;
+        text-overflow: ellipsis;
```

Comment:
These still don't do anything. I guess since we require `[tuiTitle]` and styles aren't encapsulated we might as well just drop .t-title wrapper and add `margin-inline-end: auto` directly on `> [tuiTitle]`. Also `justify-content: flex-end;` on `.t-header` in case people use it without title.